pupil-spa/src/app/check/check.component.spec.ts

```diff
@@ -8,21 +8,32 @@ import { AnswerService } from '../services/answer/answer.service';
 import { StorageService } from '../services/storage/storage.service';
 import { SubmissionService } from '../services/submission/submission.service';
 import { QuestionServiceMock } from '../services/question/question.service.mock';
-
-const mockQuestionService = new QuestionServiceMock();
+import { WarmupQuestionService } from '../services/question/warmup-question.service';
 
 describe('CheckComponent', () => {
   let component: CheckComponent;
   let fixture: ComponentFixture<CheckComponent>;
 
+  function detectStateChange(object, method, arg?) {
```

Comment:
Seems like a util function to me.
Would you rather maintain this separately in case of reusability?

Reply:
It's specific to this component and runs an expectation.. so I think it's fine where it is.  It just reduces dups.

Reply:
Fair enough